api/v1alpha1/managedcluster_types.go

```diff
@@ -28,6 +28,7 @@ const (
 	FluxHelmChartNameKey      = "helm.toolkit.fluxcd.io/name"
 	FluxHelmChartNamespaceKey = "helm.toolkit.fluxcd.io/namespace"
 
+	PartOfLabel          = "hmc.mirantis.com/part-of" // PartOfNameLabel label key with value containing the name of the release owner
```

Comment:
I think we can figure this out based on ownership. ManagedCluster-owned helmreleases have an OwnerReference set already. We can assume for now, that helmreleases without ownerreferences belong to providers (and maybe set ORs for them later)

Reply:
Thanks, fixed it with a simple length check and left a simple check just in case (noop because of the length check)